lang/en.json

```diff
@@ -521,135 +521,154 @@
-        "PageRenderTimeVariableName": "Page Render Time",
-        "PageTitleVariableDescription": "Returns the title of the current page.",
-        "PageTitleVariableName": "Page Title",
-        "PageUrlVariableDescription": "Returns the full URL of a page",
-        "PageUrlVariableName": "Page URL",
-        "PageViewTriggerDescription": "Triggered as soon as the Tag Manager is executed within the page.",
+        "PageLoadTimeTotalVariableName": "Total page-load time",
+        "PageOriginVariableDescription": "Returns a domain of the current URL with its protocol.",
+        "PageOriginVariableName": "Web-page origin",
+        "PagePathVariableDescription": "Returns the path of the current web page URL",
+        "PagePathVariableName": "Web-page path",
+        "PageRenderTimeVariableDescription": "The amount of milliseconds it took to render the page in the web browser.",
+        "PageRenderTimeVariableName": "Web-page render time",
+        "PageTitleVariableDescription": "Returns the title of the current web page.",
+        "PageTitleVariableName": "Web-page title",
+        "PageUrlVariableDescription": "Returns the full URL of a web page",
+        "PageUrlVariableName": "Web-page URL",
+        "PageViewTriggerDescription": "Triggered as soon as the tag manager is executed within the web page.",
         "PageViewTriggerName": "Pageview",
-        "PingdomRUMTagDescription": "Pingdom Real User Monitoring (RUM) lets you collect performance data from actual visitors to your site.",
-        "PingdomRUMTagHelp": "This tag allows you to add the Pingdom Real User Monitoring (RUM) to your site.",
+        "PingdomRUMTagDescription": "Pingdom real user monitoring (RUM) lets you collect performance data from actual visitors of your web site.",
+        "PingdomRUMTagHelp": "This tag allows adding the Pingdom real user monitoring (RUM) to your website.",
         "PluginDescription": "Manage and unify all your tracking and marketing snippets in one place.",
         "PreConfiguredInfoTitle": "Pre-configured variables are variables that are ready-to-use and cannot be created, edited, or configured. They work out of the box and you can use them in tags, triggers, and variables if needed.",
         "PreconfiguredVariables": "Pre-configured variables",
-        "PreviewDebugEnabledNotificationLine1": "Preview mode is enabled%1$s. Access your website or enter a URL to debug the container now. ",
-        "PreviewDebugEnabledNotificationLine2": "If you want to %1$sshare the preview%2$s with someone else, please append %3$s or %4$s to the URL of your website.%5$sWhile this mode is enabled, the preview container will be automatically updated when you make a change.",
-        "PreviewDebugEnabledNotificationLine3": "Debug not working? Please check this %1$sFAQ%2$s.",
+        "PreviewDebugEnabledNotificationLine1": "Preview mode is on%1$s. Access your website or enter a URL to debug the container now. ",
```

Comment:
enabled as earlier sounds more appropriate in comparison to `on`

Reply:
The idea is to never use enable/d or disable/d,
unless something can be on, and still disabled when that
distinction is also meaningful to communicate.

https://hosted.weblate.org/languages/nb_NO/matomo/search/?offset=1&q=enabled&sort_by=-priority%2Cposition&checksum=
are the ones left (most of which are here)